tools/autopep8_extended.py

```diff
@@ -20,8 +20,29 @@ def __init__(self, pep8_options, source):
                       'should use CamelCase. Change of "{old_class_name}" '
                       'to "{new_class_name}" next (lines,columns): '
                       '{lines_columns}',
+            'CW0002': 'VIM comment found',
         }
 
+    def check_cw0002(self):
+        'Detect vim comment'
+        msg_code = 'CW0002'
+        check_result = []
+        msg = self.msgs[msg_code]
+        line_no = 0
+        for line in self.source:
+            column = 0
+            line_no += 1
+            line = line.strip()
+            if line.startswith('#') and \
```

Comment:
In this case regex is a good idea... line.startwith too.
I saw a post of problems with regex recently...
I prefered avoid them in this simple case.